ci-operator/config/openshift-priv/frr/openshift-priv-frr-master.yaml

```diff
@@ -48,8 +48,6 @@ tests:
         NETWORK_TYPE=OpenShiftSDN
       E2E_TESTS_CONFIG: |
         IP_STACK=v4
-        METALLB_BRANCH=main
```

Comment:
these are not changed by us. We normally change only ci-operator/config and run make jobs.

Reply:
I mean, we normally touch only config/openshift/ and not openshift-priv/

Reply:
oh ok. what is it for anyway? I'll fix it together with the commit title.

Reply:
and https://github.com/openshift/release/commit/7e5ad3a10b5fea19a407268ef4ff3101f2f99f66